Content.Client/UserInterface/Systems/Sandbox/SandboxUIController.cs

```diff
@@ -164,7 +166,7 @@ public void OnStateExited(GameplayState state)
     {
         if (_window != null)
         {
-            _window.Dispose();
+            _window.Close();
```

Comment:
I tested everything and it works as intended but this is the only part I got no clue about if it is correct, and neither has sloth. I gave PJB a ping to figure this out.
The rest of the changes look good to me.

Reply:
It's a bunch of default buttons, this is fine.